tensorflow/compiler/xla/python/py_client.cc

```diff
@@ -736,12 +737,7 @@ StatusOr<std::pair<XlaOp, pybind11::object>> PyClient::EmitPythonCallback(
 XLA_CPU_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM("xla_python_cpu_callback",
                                              &XlaPythonCpuCallback);
 
-#if TENSORFLOW_USE_ROCM
 XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM("xla_python_gpu_callback",
-                                         &XlaPythonGpuCallback, "ROCM");
-#elif GOOGLE_CUDA
-XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM("xla_python_gpu_callback",
-                                         &XlaPythonGpuCallback, "CUDA");
-#endif  // TENSORFLOW_USE_ROCM
+                                         AssertionCustomCall,absl::AsciiStrToUpper(PlatformUtil::CanonicalPlatformName("gpu").value()));
```

Comment:
May I ask why we can't use ```PlatformUtil::GetDefaultPlatform().value()``` ? it returns an upper case platform name (either ROCM or CUDA) so you don't need ```absl::AsciiStrToUpper()```